man/man1/p11sak.1.in

```diff
@@ -432,3 +437,40 @@ prints help for the usage of
 .B p11sak
 and/or the respective command.
 .PP
+.
+.
+.
+.SH "p11sak_defined_attrs.conf"
+In the new output config file a user can define additional attributes, which are not 
+mentioned in the PKCS#11 standard. A custom path can be set with the environment variable
+P11SAK_DEFAULT_CONF_FILE. If none is set p11sak will first look for the file in the user 
```

Comment:
That one should also be described in a section called `ENVIRONMENT`.

Reply:
Done